package.json

```diff
@@ -25,6 +25,11 @@
   "author": "devsupport@webex.com",
   "license": "MIT",
   "dependencies": {
+    "@webex/components": "^1.18.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.10.2",
```

Comment:
I understand that this are peer dependencies of `@webex/components`, but we only really need it for testing, would it make more sense to add them as `devDependencies`?

Reply:
Sure